.travis.yml

```diff
@@ -1,7 +1,26 @@
 language: node_js
 
 node_js:
-  - 4.8.6
+  - node
+
+branches:
+  except:
+  - /^v?\d+\.\d+\.\d+$/
 
 before_install:
- - yarn config set ignore-engines true
\ No newline at end of file
+  # If package.json major and minor versions match last tag, then increment last tag. Else use package.json major.minor.0.
+  - "{ sed -nE 's/^[ \\t]*\"version\": \"([0-9]{1,}\\.[0-9]{1,}\\.)[0-9x]{1,}\",$/\\1/p' package.json; git describe --abbrev=0 | sed -E 's/^v([0-9]{1,}\\.[0-9]{1,}\\.)([0-9]{1,})$/\\1 \\2/g'; } | tr \"\\n\" \" \" | awk '{printf($1==$2?\"v\"$2$3+1:\"v\"$1\"0\")}' | xargs -I {} git tag -a {} -m \"{}\"\n"
+  - npm --no-git-tag-version version from-git
+  - yarn config set ignore-engines true
+
+deploy:
```

Comment:
I don't see npm pack command somewhere in this file?

Reply:
No, it's a bit of magic, travis should do it automatically as we're using the travis 'npm' provider, but I don't know if it will work till we land this and I test the build.

https://docs.travis-ci.com/user/deployment/npm/